pkg/security/probe/syscall_stats.go

```diff
@@ -160,8 +225,25 @@ func NewSyscallMonitor(manager *manager.Manager) (*SyscallMonitor, error) {
 		return nil, errors.New("map noisy_processes_bb not found")
 	}
 
+	execFrontBuffer, ok, err := manager.GetMap("exec_count_one")
+	if err != nil {
+		return nil, err
+	}
+	if !ok {
+		return nil, errors.New("map exec_count_one not found")
+	}
+
+	execBackBuffer, ok, err := manager.GetMap("exec_count_two")
```

Comment:
Could you rename `exec_count_two` to `exec_count_bb`?

Reply:
👍